docs/configuration.md

```diff
@@ -1954,6 +1954,46 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 </table>
 
+### Thread Configurations
+
+Depending on jobs and cluster configurations, we can set number of threads in several places in Spark to utilize 
+available resources efficiently to get better performance. Prior to Spark 3.0, these thread configurations apply 
+to all roles of Spark, such as driver, executor, worker and master. From Spark 3.0, we can configure threads in 
+finer granularity starting from driver and executor. Take RPC module as example in below table. For other modules,
+like shuffle, just replace "rpc" with "shuffle" in the property names except 
+spark.{driver|executor}.rpc.netty.dispatcher.numThreads, which is only for RPC module.
```

Comment:
Wrap config names with `<code></code>`, here and elsewhere.

Reply:
@vanzin , updated, please check.